### PR TITLE
added new docs for local Camera switcher button

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What
Adds docs for the local camera switcher button.

# Why

Provides documentation about the new button on the local video tile in the videoGallery

https://skype.visualstudio.com/SPOOL/_workitems/edit/2698414
# How Tested

ran local version of storybook.

### CODE REVIEW

**Review by alkwa-msft — APPROVED** [review]

File: change/@internal-storybook-55fe21fd-776e-4439-a147-c11615ec0b90.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```
Comment (JamesBurnside) [inline change/@internal-storybook-55fe21fd-776e-4439-a147-c11615ec0b90.json:2]: prerelease

File: packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx [hunk 2]
```diff
@@ -118,6 +120,17 @@ const getDocs: () => JSX.Element = () => {
         <CustomStyleVideoGalleryExample />
       </Canvas>
 
+      <Heading>Local Video Camera Button</Heading>
+      <DetailedBetaBanner />
+      <Description>
+        The Video Gallery can take in customization to allow for the introduction of local camera controls where the
+        button will cycle through the different camera's in the users device. Typical usage is to trigger this button on
```
Comment (JamesBurnside) [inline packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:127]:
```suggestion
        button will cycle through the different camera's in the users device. Typical usage is to enable this button on
```

File: packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx [hunk 3]
```diff
@@ -118,6 +120,17 @@ const getDocs: () => JSX.Element = () => {
         <CustomStyleVideoGalleryExample />
       </Canvas>
 
+      <Heading>Local Video Camera Button</Heading>
+      <DetailedBetaBanner />
+      <Description>
```
Comment (JamesBurnside) [inline packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:125]: mention the prop that can be used to enable it in the description

File: packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx [hunk 4]
```diff
@@ -118,6 +120,17 @@ const getDocs: () => JSX.Element = () => {
...
+        The Video Gallery can take in customization to allow for the introduction of local camera controls where the
+        button will cycle through the different camera's in the users device. Typical usage is to trigger this button on
+        mobile devices.
+      </Description>
+      <Canvas>
```
Comment (alkwa-msft) [inline packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:130]: did we not want to post the source as well?

**Review by JamesBurnside — APPROVED** [review]

File: packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx (hunk 3 above)
Comment (dmceachernmsft) [reply packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:125]: okay will add that!

File: packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx (hunk 4 above)
Comment (dmceachernmsft) [reply packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:130]: I'm not sure what you mean? of where the gifs came from?

Comment (dmceachernmsft) [reply packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx:130]: sorry just figured it out. will add it!

**Pull request merged**